rust/connlib/tunnel/src/client.rs

```diff
@@ -828,13 +828,14 @@ impl ClientState {
             .map(|(_, res)| res.id)
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
     fn update_system_resolvers(&mut self, new_dns: Vec<IpAddr>) -> bool {
         if !dns_updated(&self.system_resolvers, &new_dns) {
             tracing::debug!("Updated dns called but no change to system's resolver");
             return false;
         }
 
-        tracing::info!("Found new system resolvers: {new_dns:?}");
+        tracing::info!("Found new system resolvers");
```

Comment:
If the resolvers shouldn't be logged, then this should change, too https://github.com/firezone/firezone/blob/chore/fine-tune-log-levels/rust/gui-client/src-tauri/src/client/resolvers.rs#L45

Reply:
I left that one in because it is at the `debug` level